package/MDAnalysis/analysis/pca.py

```diff
@@ -247,7 +247,8 @@ def __init__(self, universe, select='all', align=False, mean=None,
 
     def _prepare(self):
         # access start index
-        self._u.trajectory[self.start]
+        # self._u.trajectory[self.start]
+        self._sliced_trajectory[0]
```

Comment:
I could be going insane but do we not need to pull the first index from `frames` or is that already handled? @orbeckst ?

Reply:
Not sure I fully understand your question I think I had a similar thought but unsure - do you mean this? https://github.com/MDAnalysis/mdanalysis/blob/846131cb4a16cea48f3ccc56dce9a82995c8be27/package/MDAnalysis/analysis/base.py#L357-L366

Reply:
Yes sorry, poorly worded by me but you read my mind. Indeed it is safe to take the 0th index as the trajectory is pre-sliced on L366.

Reply:
Thanks @IAlibay!

Reply:
So my question here is whether or not this behaviour is defined anywhere. There's a versionchanged somewhere but quickly scanning the docstring there's no info on this right? (maybe I'm missing it somewhere)

Reply:
This behaviour == the behaviour in this analysis class to take the first frame for the mean selections.

Reply:
@IAlibay I could be interpreting this wrong but it seems like this fixes a bug to make things behave the way they should no? It is just enforcing consistent slicing behaviour between the different ways of slicing the trajectory?

Reply:
Please don't take my comment here as "this isn't the right fix", but rather "should there be better documentation" (which doesn't necessarily have to be handled here but if it's a few lines it could?)

To explain how I got here - I started reviewing, got to the code and went "ok makes sense", then looked around for docs to make sure it was the right behaviour and couldn't find it.

Reply:
Ah I see what you mean. Perhaps adding the below to the docs would make it clearer? 
```
When using mean selections, the first frame of the selected trajectory slice is used as a reference.
```

Reply:
Yeah I think so? I'm probably missing something really obvious somewhere though.

Also as mentioned above, this can just be a separate issue, we don't have to fix it here if @marinegor doesn't feel like it.